src/components/05_pages/Content/Content.js

```diff
@@ -272,14 +272,10 @@ class Content extends Component {
                     >
                       <EditIcon />
                     </IconButton>
-                    <IconButton
-                      aria-label="delete"
-                      className={styles.button}
-                      component={Link}
-                      to={`/node/${nid}/delete`}
-                    >
-                      <DeleteIcon />
-                    </IconButton>
+                    <DeleteButton
```

Comment:
We should make sure that current styles still apply for the new icon. It seems like the positioning and the color is a bit off.

<img width="514" alt="screen shot 2018-06-26 at 18 01 17" src="https://user-images.githubusercontent.com/1845495/41907663-c2e571f8-796b-11e8-9716-a1247e8bfc1e.png">